edxval/models.py

```diff
@@ -605,6 +605,55 @@ def __unicode__(self):
         return u'{course_id} - {provider}'.format(course_id=self.course_id, provider=self.provider)
 
 
+class OrganizationTranscriptCredentialsState(models.Model):
```

Comment:
Also, we should probably use `TimeStampedModel` to keep track of created and modified time